bigtable/hello/main.py

```diff
@@ -27,13 +27,17 @@
 """
 
 import argparse
-import uuid
 
 from gcloud import bigtable
 from gcloud.bigtable import happybase
 
 
 def main(project, cluster_id, zone, table_name):
+    # [START connecting_to_bigtable]
+    # project = "my-project-id"
```

Comment:
I agree w/ Jon - rename project => projectId (or the Python convention), same for the others.